Recruit-Training/Embedded-Recruit-Training/Core/Src/main.c

```diff
@@ -288,6 +328,41 @@ void HAL_CAN_RxFifo0MsgPendingCallback(CAN_HandleTypeDef* hcan)
     }
 
     //TODO: Match StdId of header and data length content for green and blue messages and check data for set bit of toggling green and blue led
+    if ((hdr.StdId == BLUE_MESSAGE_STDID || hdr.StdId == GREEN_MESSAGE_STDID) && hdr.DLC == 1)
+    {
+        if (data[0] == 1 && data[3] == 1 && data[7] == 1)
```

Comment:
I think we mentioned last time that this should be a bitwise operation to check which bits has been set.
To check if bits 0, 3 and 7 is set you can set a bitmask like `10001001`. That way when it's bitwise AND ( `&`) together only bits 0, 3 and 7 are going to be a 1 if they were a 1 before. Since `0 & 0 = 0`, `1 & 0 = 1`, `1 & 1 = 1`.